L1Trigger/L1TZDC/plugins/L1TZDCProducer.cc

```diff
@@ -0,0 +1,4885 @@
+                                            78945.8,
+                                            65279,
+                                            66658.4,
+                                            336349,
+                                            191719,
+                                            124449,
+                                            141267,
+                                            45079.8,
+                                            79740.3,
+                                            72503.4,
+                                            62236.7,
+                                            44388.7,
+                                            336349,
+                                            215263,
+                                            100905,
+                                            94177.7},
+                                           {35648.5,
+                                            80242.1,
+                                            81966.6,
+                                            67776.8,
+                                            69209,
+                                            349219,
+                                            199055,
+                                            129211,
+                                            146672,
+                                            46804.7,
+                                            82791.4,
+                                            75277.6,
+                                            64618.1,
+                                            46087.2,
+                                            349219,
+                                            223500,
+                                            104766,
+                                            97781.3}};
+
+  //inputs
+  Handle<QIE10DigiCollection> zdcDigiCollection;
+  iEvent.getByToken(m_zdcToken, zdcDigiCollection);
+
+  //In lieu of bxFirst, bxLast, use the number of the timeslice samples
+  const QIE10DataFrame& frametest = (*zdcDigiCollection)[0];
+  int nSamples = frametest.samples();
+  //outputs
+
+  EtSumBxCollection etsums(0, 0, nSamples);
+  EtSumBxCollection etsumsReduced(0, 0, nSamples);
+
+  //We need to reduce to 5 bunches
+  const int peakBX = 4;
+  const int bxRPlusMinus = 2;
+  const int startBXR = peakBX - bxRPlusMinus;
+  const int endBXR = peakBX + bxRPlusMinus;
+
+  //rawadc[detector index][time slices]
+  unsigned short rawadc[18][10];
+
+  int counter = 0;
+  // the loop below loops over all the elements of the QIE10DigiCollection. Each entry corresponds to one channel
+  for (QIE10DigiCollection::const_iterator it = zdcDigiCollection->begin(); it != zdcDigiCollection->end(); it++) {
+    const QIE10DataFrame& frame(*it);
+    HcalZDCDetId cell = frame.id();
+    int zside = cell.zside();
+    int section = cell.section();
+    int channel = cell.channel();
+    //FIXME: removing non ZDC channels? TODO check.
```

Comment:
Does this need to be fixed?

Reply:
To my knowledge no, we get perfect agreement with offline calculations so we think the ZDC EtSum is validation; likely this is leftover, removed in latest commit